packages/client-python/src/rocketride/core/runtime/process.py

```
@@ -0,0 +1,178 @@
+    Returns the process pid.
+
+    Uses subprocess.Popen (not asyncio) so the child process is fully
+    independent of the parent — it survives after the CLI exits.
+    """
+    log_dir = logs_dir(instance_id)
+    log_dir.mkdir(parents=True, exist_ok=True)
+
+    stdout_log = log_dir / 'stdout.log'
+    stderr_log = log_dir / 'stderr.log'
+
+    stdout_fh = open(stdout_log, 'w')
```

Ruff (0.15.7)

[warning] Async functions should not open files with blocking methods like `open`

(ASYNC230)